acceptance-tests/src/test/resources/YubiShellSimulator.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
is this a duplicate from somewhere? Or is this the only one?
If its the only one, that's awesome, but if we duplicated it, might be nice to work out how to work around that

Reply:
Not duplicated ... its very dumbed-down simulator that will only work for Web23Signer ATs